x/feemarket/keeper/params.go

```diff
@@ -65,11 +71,15 @@ func (k Keeper) GetBaseFee(ctx sdk.Context) *big.Int {
 		// try v1 format
 		return k.GetBaseFeeV1(ctx)
 	}
-
 	return baseFee
 }
 
-// SetBaseFee set's the base fee in the paramSpace
+// SetBaseFee set's the base fee in the store
 func (k Keeper) SetBaseFee(ctx sdk.Context, baseFee *big.Int) {
-	k.paramSpace.Set(ctx, types.ParamStoreKeyBaseFee, sdkmath.NewIntFromBigInt(baseFee))
+	params := k.GetParams(ctx)
+	params.BaseFee = sdk.NewInt(baseFee.Int64())
```

Comment:
you can create an `sdk.Int` with a `big.Int`
```suggestion
	params.BaseFee = sdk.NewIntFromBigInt(baseFee)
```